### DISCUSSION TITLE
Not available in India App Store

### DISCUSSION THREAD

**Discussion by akshaybharwani** [pixelspark/sushitrain#2] [Q&A]
As the title says. Is there a reason for it? Would love to give this app a try!

**Comment by pixelspark** [comment]
Hi @akshaybharwani, thanks for your interest in the app. Unfortunately several countries have specific (legal) requirements on the App Store (i.e. related to encryption). I have to sort these out on a country by country basis. Perhaps the Testflight version is of use for you:  https://testflight.apple.com/join/2f54I4CM

**Reply by akshaybharwani** [reply]
Ah, totally make sense. Thanks for the Testflight link! 
good luck with the app!
